conftest.py

```diff
@@ -126,8 +126,13 @@ def s3_scratch(s3):
 
 
 @pytest.fixture(scope="function")
-def s3_url(s3, s3_scratch, request):
-    url = f"{s3_scratch}/{request.node.originalname}-{uuid.uuid4().hex}"
+def s3_url(s3, s3_scratch, test_name):
+    url = f"{s3_scratch}/{test_name}-{uuid.uuid4().hex}"
     s3.mkdirs(url, exist_ok=False)
     yield url
     s3.rm(url, recursive=True)
+
+
+@pytest.fixture
+def test_name(request):
+    return request.node.originalname
```

Comment:
I believe this is not the test name but the test file/module. Should we rename the fixture to match this?

Reply:
Good point. What this is depends on the context in which the fixture is used. If used directly in a test, it will be the test name but, as you mentioned, if called from a module scoped fixture I think it will be module name instead. I've gone ahead and reverted this change as it's not needed to add the spinup time benchmark